package.json

```diff
@@ -44,6 +43,7 @@
     "gulp-sourcemaps": "^2.6.4",
     "gulp-tap": "^0.1.3",
     "mocha": "^5.0.4",
+    "node-sass": "^4.12.0",
```

Comment:
Maybe it makes sense to get this package out of dependencies and install node-sass/sass manually?

Reply:
It's a dev dependency and won't be installed when you add `gulp-sass` in your project.

Reply:
No, it installs `node-sass` and runs the compilation, even if I don't need it and I use `sass`.